src/commands/user.ts

```diff
@@ -4,7 +4,8 @@ import { MessageEmbed } from 'discord.js';
 
 export const userCommand: CommandDef = {
   prefix: 'user',
-  description: "Get data on a current user. Use the syntax 'user [username]'.",
+  description: 'Get data on a given user, or the command caller themselves.',
```

Comment:
This is incorrect - the `user` command will not work if a user is not provided. 
This was done intentionally, as the `user` command is currently intended to tell a moderator if a user has previously been suspended. There is no need for a moderator to check this information on their own username :)

Reply:
Owh, so this is not like the user command that gives information about the user.

Reply:
Fixing it

Reply:
Yeh - it may be refactored later, but currently it only gets the prior suspension data.